app/models/chargeback_rate_detail.rb

```diff
@@ -195,6 +198,10 @@ def ambiguous_tier?(tier)
     tier.start == tier.finish
   end
 
+  def start_great_than_finish?(tier)
```

Comment:
yes, I like tier_continuity? and I moved it to model, also I removed ambiguous_tier? method beucase it is covered by tier_continuity?, ok ?

thanks @imtayadeway